src/Common/test-runtime/project.json

```diff
@@ -3,41 +3,21 @@
     "Microsoft.NETCore.Platforms": "1.0.1-rc4-24213-04",
     "Microsoft.NETCore.TestHost": "1.0.0-rc4-24131-00",
     "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2-rc4-24213-03",
-    "NETStandard.Library": "1.6.0-rc4-24213-04",
```

Comment:
It does pull down a fair amount of dependencies but it was my expectation that most of them were needed. If that isn't the case then I'm fine with trimming it out. What I don't want to happen though is that one of the xunit or TestSuite packages pulls it in at a different likely earlier version because that will just cause problems later when we try to update.